packages/headless/project.json

```diff
@@ -18,7 +18,7 @@
       ],
       "executor": "nx:run-commands",
       "options": {
-        "commands": ["npm run build:prod", "npm run build:doc"],
```

Comment:
Removing that for now because there's currently an issue with tags with the previous documentation generation mechanism that prevents builds from working.
Will fix when I work on annotations https://coveord.atlassian.net/browse/KIT-3422